<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="swimmerLocationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Location of Swimmer</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="swimmerLocationTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Location of Swimmer in the Map</string>
    </property>
   </widget>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>321</width>
      <height>200</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="alertImageLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Rescued!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
